pom.xml

```diff
@@ -158,14 +158,14 @@
         <abdera.osgi.version>1.1.3_2</abdera.osgi.version>
         <abdera.version>1.1.3</abdera.version>
         <antlr.version>4.3</antlr.version>
-        <apache.shiro.version>1.4.0</apache.shiro.version>
+        <apache.shiro.version>1.4.2</apache.shiro.version>
         <asm.version>3.3.1</asm.version>
         <awaitility.version>3.1.5</awaitility.version>
         <aws-java-sdk-bundle.version>1.11.344_1</aws-java-sdk-bundle.version>
         <bouncy.version>1.61</bouncy.version>
         <camel.version>2.24.2</camel.version>
         <cesium.version>1.34</cesium.version>
-        <common-compress.version>1.18</common-compress.version>
+        <common-compress.version>1.19</common-compress.version>
```

Comment:
❓Why was this updated?

Reply:
This is a small win while I'm in here. This is truly a one liner that resolves some CVEs pointed out in Snyk. I'm putting this in master, figured I'd put it in the 2.21.x branch while I'm here.